### PR TITLE
Check in TF 1.13.1

### PR DESCRIPTION
**Pull request by jc2729**
**What this PR does / why we need it**: Check-in TF v1.13.1 for source protos to be used in TF2OpenAPI

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/181)

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -13,14 +13,21 @@ required = [
...
     ]
 
 [prune]
   go-tests = true
   unused-packages = true
   non-go = true
 
+[[prune.project]]
+  name = "github.com/tensorflow/tensorflow"
+  non-go = false
```
Comment (ellis-bigelow) [inline Gopkg.toml]: Isn't this true?

Comment (jc2729) [reply Gopkg.toml]: I ran `dep ensure` with&without certain fields in [[prune.project]], and with&without [[prune.project]]. `dep ensure` seems to ignore [[prune.project]] completely and cares only about the required array. Removing the [[prune.project]] section

**Pull request merged**